noodles-editor/src/noodles/noodles.tsx

```diff
@@ -455,12 +515,10 @@ export function getNoodles(): Visualization {
 
       // Update URL query parameter with project name
       if (name) {
-        const url = new URL(window.location.href)
-        url.searchParams.set('project', name)
-        window.history.replaceState({}, '', url.toString())
+        navigate(`/examples/${name ?? ''}`, { replace: true })
```

Comment:
setProjectName a couple lines above calls navigate too, so would this be duplicate logic?